common/geo/srs.js

```diff
@@ -0,0 +1,41 @@
+const R = require('ramda')
+
+const keys = {
+  code: 'code',
+  name: 'name',
+  wkt: 'wkt',
+}
+
+const newSrs = (code, name, wkt) => ({
+  [keys.code]: code,
+  [keys.name]: name,
+  [keys.wkt]: wkt,
+})
+
+//EPSG:4326 WGS84 Lat Lon Spatial Reference System
+const latLonSrs = newSrs('4326', 'GCS WGS 1984', 'GEOGCS["WGS 84",\n' +
```

Comment:
I think it might be more readable with new js strings `` so you won't need the + and the \n

Reply:
right